modules/zivid/settings.py

```diff
@@ -2492,10 +2492,12 @@ def __str__(self):
     class Sampling:
         class Color:
             disabled = "disabled"
+            grayscale = "grayscale"
```

Comment:
It looks like the color settings ended up in the wrong commit?

Reply:
Yeah, I'm gonna fix it when we start merging. Before merging all of these PRs I'll make a PR to switch over to the new SDK and update the data models